src/components/annotations/AnnotationVariantItem.vue

```diff
@@ -51,75 +56,142 @@ const props = withDefaults(defineProps<Props>(), {
   toggle: () => null,
 })
 
-let variantItemsSelection = reactive({})
-let variantItemsColors = {}
-
+let initVariantItemsSelection = {}
```

Comment:
It is strange that this variable has the same name as utils function. What is the job of this variable anyway? Can this be a prop?